Message-Id: <a0de06625d26cfcf94ddd1b73a6ff83867433dd3.1336068799.git.agx@sigxcpu.org>
In-Reply-To: <32e4c48a65f59ab8c3868ab05bed1a7b98811908.1336068799.git.agx@sigxcpu.org>
References: <5909887366833c92178e3ed650b87d666f9f50c1.1336068799.git.agx@sigxcpu.org>
	<56db9d711b7615ece3848c57f71f362f0e5f4fea.1336068799.git.agx@sigxcpu.org>
	<727a1c60fdc68e24b2f457f5b7e3611b2aae7a7b.1336068799.git.agx@sigxcpu.org>
	<af426bb880140c2941366a3a84fd11a714b9c778.1336068799.git.agx@sigxcpu.org>
	<64360f6da8f2d82618e0eda5d971ee6a083851ae.1336068799.git.agx@sigxcpu.org>
	<308046ae1d09cc7cce80fdb268cfb7fee35de13d.1336068799.git.agx@sigxcpu.org>
	<f96b7f473f61a8e6d4866a49d9fcc861bed443b5.1336068799.git.agx@sigxcpu.org>
	<7e773c9b5d95a42321be6b5fc7a11cf0fd51a124.1336068799.git.agx@sigxcpu.org>
	<6d5763889d5f114ed6f149ecd47c562f06868519.1336068799.git.agx@sigxcpu.org>
	<e82f2035cd9341f3530199e68c11a27790f041de.1336068799.git.agx@sigxcpu.org>
	<87e0b605aa093d95c507083de601261098cd1aff.1336068799.git.agx@sigxcpu.org>
	<d7d5684690a5110e57b4f5a5ba48b76aff0f183c.1336068799.git.agx@sigxcpu.org>
	<574c21aa539700de7e6214f0123644312d1724b3.1336068799.git.agx@sigxcpu.org>
	<414a2a09a0c8f8b00c5804b2d5afd2ae704fd96f.1336068799.git.agx@sigxcpu.org>
	<bed21802c06c687175cf38a4a62f586a9945d804.1336068799.git.agx@sigxcpu.org>
	<790f6e636b6caf5fa02e645ea0bc473892a0b86e.1336068799.git.agx@sigxcpu.org>
	<6e9717dcb665e73ae0c596d35f77e4c60db42f5d.1336068799.git.agx@sigxcpu.org>
	<aa933d6bee9818778dd49d18cac78f61e237ae18.1336068799.git.agx@sigxcpu.org>
	<54674a84ec4ad50d943b33191aa1ffa48f35e749.1336068799.git.agx@sigxcpu.org>
	<37a22afe7d88ed8395011dc0c01de7bb8671a40d.1336068799.git.agx@sigxcpu.org>
	<601f56221d0e98cb33e7d4cec1a5d37586e4a64a.1336068799.git.agx@sigxcpu.org>
	<66a30ba49e8b45c80dfde1f2921988ce1964b97f.1336068799.git.agx@sigxcpu.org>
	<7157c037befeb30f9ae8b92232ecd4e2639d166a.1336068799.git.agx@sigxcpu.org>
	<a60d095ea97fbf436597b4bb0e19802364e60a3c.1336068799.git.agx@sigxcpu.org>
	<32e4c48a65f59ab8c3868ab05bed1a7b98811908.1336068799.git.agx@sigxcpu.org>
From: =?UTF-8?q?Guido=20G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sat, 2 Apr 2011 10:37:16 +0200
Subject: Brand pt-PT

---
 pt-PT/locale/pt-PT/lightning/lightning.dtd        |    2 +-
 pt-PT/locale/pt-PT/lightning/lightning.properties |   17 +++++++++++++----
 2 files changed, 14 insertions(+), 5 deletions(-)

diff --git a/pt-PT/locale/pt-PT/lightning/lightning.dtd b/pt-PT/locale/pt-PT/lightning/lightning.dtd
index c5fb47a..695b4d8 100644
--- a/pt-PT/locale/pt-PT/lightning/lightning.dtd
+++ b/pt-PT/locale/pt-PT/lightning/lightning.dtd
@@ -46,7 +46,7 @@
   -->
 
 <!-- Tools menu -->
-<!ENTITY lightning.taskLabel "Lightning">
+<!ENTITY lightning.taskLabel "Iceowl">
 
 <!-- New menu popup in File menu -->
 <!ENTITY lightning.menupopup.new.event.label    "Evento…">
diff --git a/pt-PT/locale/pt-PT/lightning/lightning.properties b/pt-PT/locale/pt-PT/lightning/lightning.properties
index 15e0d27..1a0780d 100644
--- a/pt-PT/locale/pt-PT/lightning/lightning.properties
+++ b/pt-PT/locale/pt-PT/lightning/lightning.properties
@@ -38,12 +38,12 @@
 # ***** END LICENSE BLOCK *****
 
 # addon description:
-extensions.{e2fda1a4-762b-4020-b5ad-a41df1933103}.name=Lightning 
-extensions.{e2fda1a4-762b-4020-b5ad-a41df1933103}.description=Um calendário para o Thunderbird
+extensions.{e2fda1a4-762b-4020-b5ad-a41df1933103}.name=Iceowl
+extensions.{e2fda1a4-762b-4020-b5ad-a41df1933103}.description=Um calendário para o Icedove
 extensions.{e2fda1a4-762b-4020-b5ad-a41df1933103}.creator=Projecto Mozilla Calendar
 
-# Lightning branding
-brandShortName=Lightning
+# Iceowl branding
+brandShortName=Iceowl
 
 # Task mode title
 taskModeApplicationTitle=Tarefas
@@ -72,8 +72,17 @@ imipBarRequestText=Esta mensagem contém um convite para um evento.
 imipBarUpdateText=Esta mensagem contém uma actualização para um evento.
 imipBarAlreadyProcessedText=Esta mensagem contém um evento que já foi processado.
 imipBarReplyText=Esta mensagem contém uma resposta a um convite.
+<<<<<<< HEAD
+<<<<<<< HEAD:pt-PT/locale/pt-PT/lightning/lightning.properties
 imipBarUnsupportedText=Esta mensagem contém um evento que esta versão do Lightning não pode processar.
+=======
+imipBarUnsupportedText=Esta mensagem contém um evento que esta versão do Iceowl não pode processar.
+>>>>>>> Brand pt-PT
 imipBarProcessingFailed=O processamento da mensagem falhou. Estado: %1$S.
+=======
+imipBarUnsupportedText=Esta mensagem contém um evento que esta versão do Iceowl não consegue processar.
+imipBarProcessingFailed=Falha a processar a mensagem. Posição: %1$S.
+>>>>>>> Brand pt-BR:pt-BR/locale/pt-BR/lightning/lightning.properties
 imipAcceptInvitation.label=Aceitar
 imipCancelInvitation.label=Eliminar
 imipDeclineInvitation.label=Declinar
